### ISSUE TITLE
Signup form should validate availability of chosen username inline

### ISSUE THREAD

**Issue by seldo** [npm/newww#1635]
As focus leaves the username box:

![screenshot 2015-12-09 14 59 03](https://cloud.githubusercontent.com/assets/185893/11701614/673329b8-9e85-11e5-85ba-ecc1b0afdd14.png)

A green tick (or whatever) for "available" or a red outline for "taken". To @ncawthon, as a low priority, for specific UI treatment :-)

**Comment by ncawthon** [comment]
if we've got the validation for this field, can we has the same for Org creation?

**Comment by seldo** [comment]
Indeed! In fact it will be the same endpoint, since those share a namespace. I'll create a separate tracker for that.
